conf/nomad/presto_standalone.hcl

```
@@ -166,8 +173,15 @@ job "${nomad_job_name}" {
       }
       template {
         data = <<EOH
+%{ if minio_use_vault_provider }
+{{ with secret "${minio_vault_kv_path}" }}
+MINIO_ACCESS_KEY = "{{ .Data.data.${minio_vault_kv_access_key_name} }}"
+MINIO_SECRET_KEY = "{{ .Data.data.${minio_vault_kv_secret_key_name} }}"
+{{ end }}
+%{ else }
 MINIO_ACCESS_KEY = "${minio_access_key}"
 MINIO_SECRET_KEY = "${minio_secret_key}"
```

```suggestion
MINIO_ACCESS_KEY="{{ .Data.data.${minio_vault_kv_access_key_name} }}"
MINIO_SECRET_KEY="{{ .Data.data.${minio_vault_kv_secret_key_name} }}"
{{ end }}
%{ else }
MINIO_ACCESS_KEY="${minio_access_key}"
MINIO_SECRET_KEY="${minio_secret_key}"
```

Shouldn't these be without spacing?